OrbitGl/App.cpp

```diff
@@ -839,15 +839,16 @@ bool OrbitApp::StartCapture() {
   }
```

Comment:
third commit has 2 bug: stanzas - one before commit message and one after.

Reply:
Rearranged the commit message.